Remove forbidden arguments from docker call

penguineer opened the pull request:
Setup for the host network from inside a container is no longer allowed.

christf commented:
this will break functionality of the container.

penguineer commented:
> this will break functionality of the container.

During our fixing session on 2022-12-08 we have established, that
a) the here mentioned arguments do not work anymore. Apparently now it is not allowed to set these parameters for the host network .
b) the container works anyways, as we are using it in the FFMD Babel network.

If you have a different experience I'd like to ask you to post your working call here, as what is written in the README is not executable.

christf commented:
There is no working call. Functionality is broken since using the host network is not allowed.

penguineer commented:
> There is no working call. Functionality is broken since using the host network is not allowed.

This is exactly the call that is running on Babel GW01 and led to a working Freifunk network.

penguineer commented:
My guess is that this needs to be configured directly on the interface via sysctl.

penguineer committed:
Remove forbidden arguments from docker call

Setup for the host network from inside a container is no longer allowed.

These settings must be made via systctl configuration.

penguineer committed:
Improve formatting of required sysctl settings
